src/export-transactions.ts

```diff
@@ -212,6 +213,23 @@ async function saveTransactions(txs: TransactionParsed[], tagged = false) {
   ])
 }
 
+async function saveTransactions(txs: TransactionParsed[], tagged = false) {
+  const txsPerYear: Map<number, TransactionParsed[]> = new Map()
+
+  for(const tx of txs) {
+    const year = new Date(tx.date).getUTCFullYear()
+    const yearTxs = txsPerYear.get(year) || []
+    yearTxs.push(tx)
+    txsPerYear.set(year, yearTxs)
+  }
+
+  for (const [year, yearTxs] of txsPerYear) {
+    await saveTransactionFiles(yearTxs, tagged, year)
+  }
+
+  await saveTransactionFiles(txs, tagged)
```

Comment:
Do you need this last `saveTransactionFiles(txs, tagged)`?

Reply:
Yes! the full file is used in other scripts